contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -790,25 +790,56 @@ is_json_query(List *name)
 * ... CONTAINS(column_name, '<contains_search_condition>') ...
 * This function transform it into a Postgres AST that stands for
 * to_tsvector(pgconfig, column_name) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>'))
+* for column_list
+* to_tsvector(pgconfig, col_1) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>') OR
+* to_tsvector(pgconfig, col_2) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>') OR
+* ...
+* * to_tsvector(pgconfig, col_n) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>')
 * where pgconfig = babelfish_fts_contains_pgconfig('<contains_search_condition>')
 */
 static Node *
-TsqlExpressionContains(char *colId, Node *search_expr, core_yyscan_t yyscanner)
+TsqlExpressionContains(List *colId, Node *search_expr, core_yyscan_t yyscanner)
 {
-    A_Expr *fts;
-    Node *to_tsvector_call, *to_tsquery_call;
-    Node *result_pgconfig;
-    List *args_pgconfig;
+	Node *fts = NULL;
+	A_Expr *column_clause;
+	Node *result_pgconfig;
+	List *args_pgconfig;
+	int colSize = colId->length;
```

Comment:
Rather than colSize, this seems to be number of cols. Also use `list_length()` instead of `->length`